lib/perl/Genome/Config/AnalysisProject/Command/DiskGroupUsage.pm

```diff
@@ -0,0 +1,85 @@
+package Genome::Config::AnalysisProject::Command::DiskGroupUsage;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Config::AnalysisProject::Command::DiskGroupUsage {
+    is => 'Command::V2',
+    has => [
+        analysis_projects => {
+            is                  => 'Genome::Config::AnalysisProject',
+            doc                 => 'the analysis projects to report disk usage',
+            is_many             => 1,
+            shell_args_position => 1,
+        },
+        print_allocation_ids_for_disk_group_name => {
+            is => 'Text',
+            is_optional => 1,
+        },
+    ],
+    doc => 'report on the disk usage of analysis projects per disk group',
+};
+
+sub help_detail {
+    return <<EOHELP
+This command will take several minutes to run.
+
+Running builds will not be accounted for properly as the allocations are not complete.
+
+The output is a tab-delimited disk usage summary for each disk group used per analysis project.
+
+The columns of the output file are:
+analysis project ID, configuration item ID, subclass name, model count, build count, total KB, total base pairs, bytes per base
+
+EOHELP
+}
+
+sub execute {
+    my $self = shift;
+
+    my @headers = qw/
+        analysis_project
+        disk_group_name
+        total_kb
+    /;
+    my $writer = Genome::Utility::IO::SeparatedValueWriter->create(
+        separator => "\t",
+        headers => \@headers,
+    );
+    for my $anp ($self->analysis_projects) {
+        my %disk_group_usage;
+        for my $config_item ($anp->config_items) {
+            for my $model ($config_item->models) {
```

Comment:
If we're not breaking the data down by config item, we could loop over `$anp->models` instead.